app/controllers/catalog_controller.rb

```diff
@@ -72,7 +72,7 @@ class CatalogController < ApplicationController
     # exploded_tag_ssim indexes all tag prefixes (see IdentityMetadataDS#to_solr for a more exact
     # description), whereas tag_ssim only indexes whole tags.  we want to facet on exploded_tag_ssim
     # to get the hierarchy.
-    config.add_facet_field 'exploded_tag_ssim',               label: 'Tag',                 partial: 'blacklight/hierarchy/facet_hierarchy'
+    config.add_facet_field 'exploded_tag_ssim',               label: 'Tag',                 limit: 9999, partial: 'blacklight/hierarchy/facet_hierarchy'
```

Comment:
That's fine with me to make it `-1`. I haven't tried it out though yet. I was thinking that the tags facet in particular could have more than 9,999 values. Is there a way to count the number of facet values easily?